awx/main/models/credential.py

```diff
@@ -332,6 +336,8 @@ def validate_role_assignment(self, actor, role_definition):
 
 
 class CredentialType(CommonModelNameNotUnique):
+    CREDENTIAL_REGISTRATION_ADVISORY_LOCK_NAME = 'setup_tower_managed_defaults'
```

Comment:
in a follow up PR we might want to change the name from "setup_tower_managed_defaults"